model/godleyTableWindow.cc

```diff
@@ -414,12 +414,15 @@ namespace minsky
         selectedCol>=0 && size_t(selectedCol)<godleyIcon->table.cols())
       {
         auto& str=godleyIcon->table.cell(selectedRow,selectedCol);
-        pango.setMarkup(defang(str));
-        int j=0;
-        if (selectedCol>=int(scrollColStart)) j=selectedCol-scrollColStart+1;
-        x-=colLeftMargin[j]+2;
-        x*=zoomFactor;
-        return x>0 && str.length()>0?pango.posToIdx(x)+1: 0;
+        if (str!=godleyIcon->table.cell(1,0))                           // No text index needed for a cell that is immutable. For ticket 1064
```

Comment:
Don't you just want tp avoid (selectedRow,selectedCol)!=(1,0)? We can't assume that (1,0) is the only cell with "Initial Conditions" in it.

Reply:
Very true, using (selectedCol!=0 || selectedRow!=1) instead, thank you.